plugins/firehydrant/src/components/ServiceAnalytics/ServiceAnalytics.tsx

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import moment, { Moment } from 'moment';
+import Alert from '@material-ui/lab/Alert';
+import { Table, TableColumn, Progress } from '@backstage/core-components';
+
+const useStyles = makeStyles({
+  container: {
+    overflow: 'auto',
+    width: '100%',
+    '& h5': {
+      fontSize: '18px !important',
```

Comment:
These are a bit unfortunate perhaps - would have felt best to place a style directly on a `Typography` or similar, to make sure it's explicitly placed where it belongs and doesn't rely on the way that the resulting DOM looks based on the JSX.  But if there's no other way at the moment, short of using a different Table...

Reply:
went ahead and removed the `fontSize` styling, but did leave the width properties up for now